internals/generators/organization/index.js

```diff
@@ -85,6 +85,12 @@ module.exports = {
       message: "Does your captive portal support log out by session ID?",
       default: false,
     },
+    {
+      type: "confirm",
+      name: "remember_me",
+      message: "Does your captive portal support automatically log me in?",
```

Comment:
mm.. did I ask you to add this?

If I am not missing anything, this is not a feature of the captive portal but something we are adding on top, because we save the session of the user from which we're then able to retrieve the radius user token and log in the user automatically.

So the question should be something like: `Do you want to support automatic logins (remember me checkbox)?`